src/styles/themes/mediaQuery.ts

```diff
@@ -10,3 +10,9 @@ export const mediaQuery = {
   mobile: '@media (max-width: 699px)',
   tablet: '@media (max-width: 1023px)'
 } as const
+
+export const matchMediaQuery = {
+  desktop: '(max-width: 1920px)',
+  mobile: '(max-width: 699px)',
+  tablet: '(max-width: 1023px)'
+}
```

Comment:
상단에서 얘기했던 부분과 이어지능데
요기서  theme에 지정한 mediaquery와 useResponsive 겹치는 타입 지정이 있어서
요런식으로하면 상단 MatchMediaQuery 단언도 없앨 수 있을 것 같구,
겹치는 타입들을 theme에서 불러와서 재사용할 수 있을 것 같은데 어떠신가유 !?!!

```suggestion
export type Device = keyof typeof mediaQuery
type MaxWidth = number
export type MatchMediaQuery = {
  [key in Device]: `(max-width: ${MaxWidth}px)`
}

export const matchMediaQuery:MatchMediaQuery = {
  desktop: '(max-width: 1920px)',
  mobile: '(max-width: 699px)',
  tablet: '(max-width: 1023px)'
}
```

Reply:
[4d824c8](https://github.com/price-offer/offer-fe/pull/53/commits/4d824c8666dbe2202188f44db5b05d8595c4f747)에서 적용 하였습니다! 감사합니다!